Gemfile

```diff
@@ -173,3 +173,5 @@ group :test do
   gem "with_model", "~> 2.1.6" # Dynamically build a model within an RSpec context
   gem "zonebie", "~> 0.6.1" # Runs your tests in a random timezone
 end
+
+gem "cgi", "~> 0.3.6" # Support for the Common Gateway Interface protocol.
```

Comment:
For other reviewers:

* 3.0.5 addresses [a response-splitting vulnerability in the CGI gem](https://www.ruby-lang.org/en/news/2022/11/22/http-response-splitting-in-cgi-cve-2021-33621/) and encourages pinning it to a version > `0.3.5`
* Also, 3.0.3 addresses [a buffer overrun in the CGI gem](https://www.ruby-lang.org/en/news/2021/11/24/buffer-overrun-in-cgi-escape_html-cve-2021-41816/) 

We haven't upgraded our Ruby in [over a year](https://www.ruby-lang.org/en/news/2021/07/07/ruby-3-0-2-released/) 😱